CRM/Contribute/Tokens.php

```diff
@@ -155,22 +156,23 @@ public function alterActionScheduleQuery(MailingQueryEvent $e): void {
 
   /**
    * @inheritDoc
+   * @throws \CRM_Core_Exception
    */
   public function evaluateToken(TokenRow $row, $entity, $field, $prefetch = NULL) {
     $actionSearchResult = $row->context['actionSearchResult'];
     $fieldValue = $actionSearchResult->{"contrib_$field"} ?? NULL;
 
-    if (in_array($field, ['total_amount', 'fee_amount', 'net_amount'])) {
+    if (array_key_exists($field, $this->getPseudoTokens())) {
```

Comment:
This is just moved higher to process it first since there is no metadata for these tokens & it will enotice

Reply:
- note I will switch in a follow up to having an isPseudoToken function